### PR TITLE
Rework README structure

### PR DESCRIPTION
**Pull request by HeavyWombat**
Reorder the sections in the README and add in usage instructions and a management summary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,54 +49,92 @@ Available Commands:
...
 ## Contributing
-
-We're happy to have other people contributing to the project. If you decide to do that, here's how to: 
+We are happy to have other people contributing to the project. If you decide to do that, here's how to:
 - fork the project
 - create a new branch
 - make your changes
 - open a PR.
+
+Git commit messages should be meaningful and follow rules nicely written by [Chris Beams](https://chris.beams.io/posts/git-commit/):
```
Comment (ansrev) [inline README.md]: it should be:
> follow the rules nicely written down

File: README.md [hunk 2]
```diff
@@ -50,54 +49,92 @@ Available Commands:
...
+> 1. Limit the subject line to 50 characters
+> 1. Capitalize the subject line
+> 1. Do not end the subject line with a period
+> 1. Use the imperative mood in the subject line
+> 1. Wrap the body at 72 characters
+> 1. Use the body to explain what and why vs. how
+
+
+### Vendoring
+It seems that vendoring via `dep` is currently a mess when you need to pull in `client-go`. We use [kubegodep2dep](https://github.com/ash2k/kubegodep2dep) to help with this. It is a very nice tool, which takes the `Godep` JSON file from the Kubernetes repository on GitHub and figures out the main constraints and overrides for the `dep` TOML file.
```
Comment (ansrev) [inline README.md]: how about:
> a very nice tool that takes...

File: README.md [hunk 3]
```diff
@@ -4,30 +4,28 @@
...
-
-## Running test cases and binaries generation
-```
-make all
-```
-
-## Commands
+# Havener
+## Introducing Havener
+Convenience tool to handle tasks around [Containerized CF](https://www.pivotaltracker.com/n/projects/2192232) workloads on a Kubernetes cluster. It deployes multiple Helm Charts using a configuration file. The configuration file is used to add in a tiny amount of glue code that is sometimes needed to make things work. Under to cover, `havener` does the same calls that `helm` and `kubectl` do, nothing special. That means that at the end you have a Helm Release just like you would have using `helm` alone.
```
[the remark this answers is not in the document]
Comment (HeavyWombat) [reply README.md]: Thanks, done

File: README.md [hunk 4]
```diff
@@ -4,30 +4,28 @@
...
+## How do I get started?
+There are different ways to get `havener`. You are free to pick the one that makes the most sense for your use-case.
+- Docker Hub serves curated Docker images with `havener` as well as  `kubectl` and other important CLI tools. There are two flavours available:
+  - [Alpine based images](https://hub.docker.com/r/havener/alpine-havener/): `docker pull havener/alpine-havener`
+  - [Ubuntu based images](https://hub.docker.com/r/havener/ubuntu-havener/): `docker pull havener/ubuntu-havener`
+- Use a convenience script to download the latest release to install it in a suitable location on your local machine:
+  ```sh
+  curl -sL https://raw.githubusercontent.com/homeport/havener/master/scripts/download-latest.sh | bash
+  ```
+- Of course, you can build it from source:
```
[the remark this answers is not in the document]
Comment (HeavyWombat) [reply README.md]: Done

**Pull request merged**